kogito-editors-js/packages/boxed-expression-component/src/components/Table/Table.tsx

```diff
@@ -152,6 +154,25 @@ export const Table: React.FunctionComponent<TableProps> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [rows]);
 
+  useEffect(() => {
+    function listener(event: CustomEvent) {
+      if (event.detail.type !== "PASTE_OP" || !tableRows.current || tableRows.current.length === 0) {
```

Comment:
Can you please create an exported const for this string? I can see that it is used also in `CopyAndPasteUtils.ts` and maybe it should be defined in that file and used here.

Reply:
Fixed! Thanks :)